Added an option to AutoUV to arrange charts horizontally or vertically

Micheus opened the pull request:
It also includes suboptions to keep or arrange the charts left/center/right or
top/center/bottom related to the selection bounding box.
This is a handy option to help us manually pack the UVs.

NOTE:
- Added an option to AutoUV to arrange charts horizontally or vertically, with
  the ability for users to adjust the padding. Thanks to envelupo for the
  suggestion;

Micheus committed:
- Added an option to AutoUV to arrange charts horizontally or vertically

It also includes suboptions to keep or arrange the charts left/center/right or
top/center/bottom related to the selection bounding box.
This is a handy option to help us manually pack the UVs.
NOTE:
- Added an option to AutoUV to arrange charts horizontally or vertically, with
  the ability for users to adjust the padding. Thanks to envelupo for the
  suggestion;

Micheus committed:
- Worked on Copilot suggestions

dgud commented on plugins_src/autouv/wpc_autouv.erl:
```diff
@@ -2357,6 +2367,123 @@ move_to(Dir,We) ->
+
+%%%% Mode Help
+arrange_mode_help(Dir,State) ->
+    io:format(" "),
```
You will have to debug this, we can't print a space every time something is buggy.

Micheus replied:
> You will have to debug this, we can't print a space every time something is buggy.

e8b897cc258a679f7714116b2e2c6419db68146a this fixed the problem.

Micheus committed:
- Fixed the reason right text was not getting updated and removed the agly
  workaround.

By debuging and tracking the process it was noticed that in wings_wm (ln:1168)
the wm_event/1 receives the changed Right text, assigning it to the #win.stk...,
but the dirty() doesn't promote the statusbar redraw.
wm_event({message_right,Name,Right0}) ->
    Right = lists:flatten(Right0),
    case lookup_window_data(Name) of
    none -> ok;
    #win{stk=[#se{msg_right=Right}|_]} -> ok;
    #win{stk=[Top0|Stk]}=Data0 ->
        Top = Top0#se{msg_right=Right},
        Data = Data0#win{stk=[Top|Stk]},
        put_window_data(Name, Data),
        dirty()
    end;
